docs/client.rst

```diff
@@ -23,3 +23,10 @@ package. Source code for such OpenWrt package can be `found here`_.
 .. _OpenWrt: https://openwrt.org/
 
 You can add the whole repository as an OpenWrt feed and add package to your firmware.
+
+Configuration
+-------------
+
+* **MAX_BROKERS** ``default: 10`` Maximum number of brokers that can be handled in a single process.
```

Comment:
```suggestion
* **MAX_BROKERS** (default: 10) Maximum number of brokers that can be handled in a single process.
```
I don't see a good reason to use code formatting here.

Reply:
Okay, then that's how it's done. Do I have to change this in my pull request?

Reply:
You should be able to press a button in GH's UI to accept my suggestion.

Reply:
I don't have the button **Commit suggestion**. I will change my commit manually.

Reply:
That's odd... it always worked for me. Thanks for doing it manually!